Code/app/src/main/java/com/acash/fitmate/fragments/PartnersFragment.kt

```diff
@@ -62,6 +62,10 @@ class PartnersFragment : Fragment() {
                     startActivity(intent)
                 }
             }
+
+            override fun getItem(position: Int): Inbox {
+                return super.getItem(itemCount - 1 - position)
+            }
         }
```

Comment:
did u try setting reverseLayout parameter in layout manager as true instead of this for descending order?

Reply:
Yes, otherwise the order is reversed.

Reply:
If that works, it would be simpler to implement I guess

Reply:
I think there is not other way of sorting other than reversing layout.